content/index.ipynb

```diff
@@ -1 +1,1683 @@
+{
```

Comment:
the formatting drastically changes, making it hard to see the diff-- also, i am wondering if this is the latest file we should be modifying? with the neurolibre business happening in parallel i'm confused about the correct workflow-- @mathieuboudreau to the rescue 😅